src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -381,18 +380,13 @@ class ActivityResourceIntTest : IntegrationTestBase() {
   inner class UpdateActivity {
 
     private lateinit var courseActivity: CourseActivity
-    private lateinit var offender: Offender
     private lateinit var offenderBooking: OffenderBooking
 
     @BeforeEach
     fun setUp() {
       testData(repository) {
         programService {
-          courseActivity = courseActivity {
-            payRate()
-            courseSchedule()
-            courseScheduleRule()
-          }
```

Comment:
Default a basic pay rate, schedule and rules if they haven't been overridden for a specific test case